playground/server/api/custom/echo.ts

```diff
@@ -0,0 +1,9 @@
+import { defineEventHandler } from 'h3'
+
+export default defineEventHandler(async (event) => {
+  return {
+    method: event.node.req.method,
```

Comment:
In Nitro v3 everything will change, but for the current Nitro v2, please keep using the h3 helpers like